providers/json-binding/src/main/java/org/jboss/resteasy/plugins/providers/jsonb/JsonBindingProvider.java

```diff
@@ -61,12 +60,8 @@ public Object readFrom(Class<Object> type, Type genericType,
          return jsonb.fromJson(entityStream, genericType);
       } catch (Throwable e)
       {
-         if (LogMessages.LOGGER.isDebugEnabled()) {
```

Comment:
Not sure I get what you mean. The if block here shouldn't be needed anymore because the exception is still passed into the Message that's used withing the ProcessingException below (which in turn ends up being logged at debug level now). OK?

Reply:
I see this now: https://github.com/resteasy/Resteasy/blob/master/resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/ExceptionHandler.java#L154

Ok then